package.json

```diff
@@ -65,6 +65,7 @@
         "passport-jwt": "^4.0.1",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
+        "react-full-screen": "^1.1.1",
```

Comment:
This package is stale. Last commits were 7 months ago. It's not a lot of code, but brings in 146 dependencies. When we get to doing full-screen, I recommend that you study the [react-full-screen repo](https://github.com/snakesilk/react-fullscreen) and implement the functionality you need in our own repo.